README.md

```diff
@@ -41,11 +42,11 @@ To create a migration, execute `db-migrate create` with a title. `node-db-migrat
 
 ```javascript
 exports.up = function (db, callback) {
-  callback();
```

Comment:
you have added this again, but the indenting is now false, I'm so pedantic right now :/